WSL/install.md

```diff
@@ -51,6 +50,12 @@ Once you have installed WSL, you will need to create a user account and password
 
 We recommend following our [Best practices for setting up a WSL development environment](./setup/environment.md) guide for a step-by-step walk-through of how to set up a user name and password for your installed Linux distribution(s), using basic WSL commands, installing and customizing Windows Terminal, set up for Git version control, code editing and debugging using the VS Code remote server, good practices for file storage, setting up a database, mounting an external drive, setting up GPU acceleration, and more. 
 
+## Upgrade from WSL 1 to WSL 2
+
+To update from WSL 1 to WSL 2 on previously installed Linux distributions, use the command: `wsl --set-version <distro name> 2` replacing `<distro name>` with the name of the Linux distribution that you want to update. For example, `wsl --set-version Ubuntu-20.04 2` will set your Ubuntu 20.04 distribution to use WSL 2. New Linux installations will be set to WSL 2 by default, but the `wsl --set-version` command can be used to downgrade from WSL 2 to WSL 1 as well. To see whether your Linux distribution is set to WSL 1 or WSL 2, use the command: `wsl -l -v`.
```

Comment:
We should mention that you'll need to enable the virtual machine platform optional component, and install the kernel package to get it working as well.